setup.py

```diff
@@ -5,16 +5,19 @@
 from setuptools.extension import Extension
 from Cython.Build import cythonize
 
-cyarchive_extension = Extension(
+_libraries = ["archive_and_deps"]
+if sys.platform == 'win32':
+    _libraries.append('advapi32')
+    _libraries.append('user32')
```

Comment:
@mingwandroid Is there a better method to link system libraries.  This seemed like a kludge to me but I could figure out a better method.